tests/publiccloud/azure_cli.pm

```diff
@@ -66,6 +68,11 @@ sub run {
     # Check that the machine is reachable via ssh
     my $ip_address = script_output("az vm list-ip-addresses -g $resource_group -n $machine_name --query '[].virtualMachine.network.publicIpAddresses[0].ipAddress' --output tsv", 90);
     script_retry("ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no azureuser\@$ip_address hostnamectl", 90, delay => 15, retry => 12);
+
+    my $location="southeastasia"
```

Comment:
please write perl code which is executable , this one is not

Reply:
done